test/api/c_api_example.c

```diff
@@ -120,6 +120,192 @@ main (int argc, char **argv)
+   -2.38832182, 5.14033352, 13.50965828,
+   -3.51544014, 6.70782563, 12.73596360,
+   -3.43813825, 7.76448415, 12.47983781,
+   -4.31222592, 6.72367359, 13.47958380,
+   -3.89122424, 6.13450845, 11.88842729,
+   -1.71962790, 6.81419378, 14.64196118,
+   -2.56489369, 6.73869928, 15.34188058,
+   -1.30553616, 8.28708010, 14.46709643,
+   -2.12153254, 8.88339645, 14.05885812,
+   -0.43293661, 8.42096985, 13.82777397,
+   -0.91009506, 8.84195389, 15.83709025,
+   -0.62198472, 9.88210532, 15.68458920,
+   -1.73049863, 8.91344164, 16.56640493,
+    0.20835472, 8.06516304, 16.53450843,
+    1.11027013, 8.08379059, 15.90505974,
+    0.58406294, 8.62609252, 17.90749094,
+   -0.27017230, 8.58733716, 18.58335654,
+    0.94521765, 9.64819987, 17.79399575,
+    1.34478256, 8.03159458, 18.41331915,
+   -0.22842475, 6.60948035, 16.70992382,
+    0.63592593, 6.02069012, 17.05055754,
+   -1.04896081, 6.48967415, 17.43260784,
+   -0.57931297, 6.05802189, 15.32665341,
+    0.29823277, 6.06356948, 14.66342487,
+   -1.03049345, 4.72745117, 15.59290274,
+   -0.25892075, 4.19008917, 15.83503152};
+
+  double energy = 0;
+  
+  int    npc = 32;
+
+  // external point charges
+  double pc[32] = {0.431000,-0.664000, 0.173000, 0.020000, 0.020000, 0.020000, 
+    0.431000,-0.664000, 0.173000, 0.020000, 0.020000, 0.020000, 0.431000,-0.664000, 
+    0.173000, 0.020000, 0.020000, 0.020000, 0.431000,-0.664000, 0.173000, 0.020000, 
+    0.020000, 0.020000, 0.431000,-0.664000, 0.173000, 0.020000, 0.020000, 0.020000, 
+    0.431000,-0.664000};
+
+  // coordinates of point charges
+  double pccoord[3*32] = {
+   -0.897837, 14.879088, 29.374541,
+   -0.512003, 15.284953, 28.580010,
+   -0.503177, 16.714209, 28.534938,
+    0.232489, 17.024434, 27.792783,
+   -0.183554, 17.159648, 29.477095,
+   -1.527540, 17.009207, 28.307266,
+   13.961254, 11.508338, 12.790715,
+   14.307185, 11.803974, 11.931815,
+   13.896755, 10.872072, 10.927705,
+   14.293925,  9.878504, 11.135478,
+   12.808068, 10.885327, 10.875863,
+   14.245566, 11.169547,  9.938799,
+   13.606744,  9.874548, 15.190554,
+   13.817846,  9.667827, 14.264552,
+   14.639146,  8.517687, 14.045818,
+   15.671299,  8.703176, 14.343150,
+   14.107705,  7.702362, 14.536639,
+   14.743162,  8.350417, 12.973787,
+    6.716485, -6.346264, 18.679919,
+    5.824101, -6.265903, 19.056785,
+    5.047806, -5.789462, 17.954393,
+    5.386539, -6.101654, 16.966443,
+    4.894122, -4.710696, 17.927294,
+    4.068575, -6.261430, 18.035357,
+   -6.316725, 11.850365, 28.030715,
+   -5.539369, 11.280064, 28.154212,
+   -4.736778, 11.127486, 26.980712,
+   -4.402790, 12.131419, 26.718633,
+   -5.410452, 10.748895, 26.211847,
+   -3.938730, 10.404747, 27.150584,
+    8.135514, 15.983725, 28.508396,
+    7.989254, 16.686992, 27.853495};
+
+  int numbers[32] = {
+    99, 99, 99, 99, 99, 99, 99, 99, 99, 99, 
+    99, 99, 99, 99, 99, 99, 99, 99, 99, 99, 
+    99, 99, 99, 99, 99, 99, 99, 99, 99, 99, 
+    99, 99};
```

Comment:
The chemical hardness for element 99 (Es) is not defined with neither GFN1-xTB nor GFN2-xTB and will therefore be zero leading to an infinity and therefore removing the interaction with the external charges effectively from the calculation.

Usually we use 7 (nitrogen) as dummy element for external charges.

Reply:
> The chemical hardness for element 99 (Es) is not defined with neither GFN1-xTB nor GFN2-xTB and will therefore be zero leading to an infinity and therefore removing the interaction with the external charges effectively from the calculation.
> 
> Usually we use 7 (nitrogen) as dummy element for external charges.

I was taking the value from this page of the docs: https://github.com/grimme-lab/xtb_docs/blob/main/source/pcem.rst

For future reference, is nitrogen your general recommendation if one does not know where the point charges come from?

Reply:
They are no real point charges, rather Slater like densities parametrized with a hardness. Setting the hardness to a large value makes them point charge like.

There is a subtle issue there, we specify element symbols to get the hardness of the element, however if a number is given it is taken as hardness in Eh/e^2. In the API we don't bother dealing with strings and therefore use atomic numbers to identify elements, this leaves no room for overwriting the hardness for each element.

Recommendations are always difficult, we try to choose something that doesn't break most of the time, whatever this meant when the original implementation was done. For an actual calculation you should carefully check whether the hardnesses provided by the method work, you can use a stub hardness or better go for a point charge like behavior.